docs/getting-started/wallets.md

```diff
@@ -6,15 +6,20 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 # Wallets, Coldkeys and Hotkeys in Bittensor
 
-In Bittensor (like other cryptocurrency applications), a *wallet* is a tool for proving your identity, signing transactions, accessing your TAO, and managing your stake in subnets. This page introduces the core concepts involved.
+In Bittensor (like other cryptocurrency applications), a *wallet* is a tool for proving your identity, signing transactions, accessing your TAO, and managing your stake in subnets.
+This page introduces the core concepts involved.
 
 For detailed procedures for handling wallets and keys, see: [Working with keys](../working-with-keys.md)
 
 For detailed security considerations, see: [Coldkey and Hotkey Workstation Security](./coldkey-hotkey-security.md)
 
 ## What are wallets and keys?
 
```

Comment:
these single newlines don't break up the lines into separate paragraphs, you need a space in between. In any case, I think this should be a paragraph, it's too broken up into separate lines if that was your intent.

Reply:
Nope, my style is to put a sentence on each line. I didn't intend to break them up. It should be one thought, agreed, but it is better for reading if each sentence is on its own line.

Reply:
It's just good editing and standard at EDB for instance.